acceptance/tests/direct_puppet/cached_catalog_remediate_local_drift.rb

```diff
@@ -0,0 +1,112 @@
+test_name "PUP-5122: Puppet remediates local drift using code_id and content_uri"
```

Comment:
FYI @ all
test_names and steps should all have blocks (do end most of the time), moving forward.  this helps beaker with indentation and beaker.next with reporting and other user features.